micro-rdk-server/Cargo.toml

```diff
@@ -50,8 +50,8 @@ path = "esp32/main.rs"
 
 
 [package.metadata.esp-idf-sys]
+esp_idf_version = "v4.4.4"
 esp_idf_tools_install_dir = "global"
 esp_idf_sdkconfig_defaults = [
   "micro-rdk-server/sdkconfig.defaults",
-  "micro-rdk-server/sdkconfig.defaults.esp32"
```

Comment:
It should not be necessary to have this file since the version associated with the current MCU should be auto selected based on the basename above.